bump-version/index.ts

```diff
@@ -45,6 +45,8 @@ class BumpVersion extends Action {
 			'--exact',
 			'--yes',
 		])
+		//regenerate yarn.lock file
+		await exec('yarn')
```

Comment:
Is yarn command available? Might want to wrap it with some error handling just in case?

Reply:
> 

done